pandas/tests/io/json/test_deprecated_kwargs.py

```diff
@@ -0,0 +1,32 @@
+"""
+Tests for the deprecated keyword arguments for `read_json`.
+"""
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas.util.testing import assert_frame_equal
```

Comment:
the linter wants you to use tm.assert_frame_equal instead of importing it

Reply:
OK.